hatchet/util/perf_measure.py

```
@@ -0,0 +1,72 @@
+# Copyright 2017-2023 Lawrence Livermore National Security, LLC and other
+# Hatchet Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+
+import os
+
+try:
+    from pycaliper import annotate_function
+    from pycaliper.instrumentation import begin_region, end_region
+
+    _PYCALIPER_AVAILABLE = True
+except Exception:
+    _PYCALIPER_AVAILABLE = False
+
+try:
+    import perfflowaspect.aspect
+
+    _PERFFLOWASPECT_AVAILABLE = True
+except Exception:
+    _PERFFLOWASPECT_AVAILABLE = False
+
+
+def _validate_perf_plugin(plugin_name):
+    print("Perf Plugin =", plugin_name)
+    print("Caliper is available?", _PYCALIPER_AVAILABLE)
+    if plugin_name.lower() == "caliper" and _PYCALIPER_AVAILABLE:
+        return "caliper"
+    elif plugin_name.lower() == "perfflowaspect" and _PYCALIPER_AVAILABLE:
+       return "perfflowaspect"
+    return "none"
```

This looks like it will fail silently - what if someone wants perfflowaspect and sets the env var but the library is not installed correctly? We should probably display a helpful error message in cases like that (somewhat like how you display caliper information in your prints maybe).